backend/mvnw.cmd

```diff
@@ -43,7 +43,7 @@ title %0
 @REM set %HOME% to equivalent of $HOME
 if "%HOME%" == "" (set "HOME=%HOMEDRIVE%%HOMEPATH%")
 
-@REM Execute a user defined script before this one
+@REM Execute a appUser defined script before this one
```

Comment:
What is this "mvnw.cmd" sorcery? I need to investigate further! 👀 🧙‍♂️

Reply:
I have no idea^^never touched this file...must be MAGIC! 😮